packages/transport/test/testHelpers/AdminApiClient.ts

```diff
@@ -0,0 +1,40 @@
+import axios, { Axios } from "axios";
+import { AccountController, CoreId, IdentityDeletionProcess } from "../../src";
+import { TestUtil } from "./TestUtil";
+
+export class AdminApiClient {
+    private static adminClient: Axios | undefined;
+
+    public static async getBackboneAdminApiClient(): Promise<Axios> {
+        if (AdminApiClient.adminClient) {
+            return AdminApiClient.adminClient;
+        }
+        const adminAPIBaseUrl = process.env.NMSHD_TEST_BASEURL_ADMIN_API!;
```

Comment:
The trailing bang `!` is unnecessary since you're checking for `null`/`undefined` in the next line.